SimplePolicy/SimplePolicyPlugin.cs

```diff
@@ -63,22 +64,44 @@ private static IEnumerable<Transaction> FilterForBlock_Policy2(IEnumerable<Trans
             if (!(transactions is IReadOnlyList<Transaction> tx_list))
                 tx_list = transactions.ToArray();
 
-            Transaction[] free = tx_list.Where(p => p.IsLowPriority)
+            if (Blockchain.Singleton.GetSnapshot().Height < ProtocolSettings.Default.FreeGasChangeHeight)
+            {
+                Transaction[] free = tx_list.Where(p => p.IsLowPriority)
                 .OrderByDescending(p => p.NetworkFee / p.Size)
                 .ThenByDescending(p => p.NetworkFee)
                 .ThenByDescending(p => InHigherLowPriorityList(p))
                 .ThenBy(p => p.Hash)
-                .Take(Settings.Default.MaxFreeTransactionsPerBlock)
+                .Take(20)
+                .ToArray();
+
+                Transaction[] non_free = tx_list.Where(p => !p.IsLowPriority)
+                .OrderByDescending(p => p.NetworkFee / p.Size)
+                .ThenByDescending(p => p.NetworkFee)
+                .ThenBy(p => p.Hash)
+                .Take(479)
```

Comment:
Why 479?

Reply:
> Why 479?

20 (free) + 479 (no_free) + 1 (minerTx) = 500

Reply:
But we should use the settings https://github.com/neo-project/neo-modules/pull/336/files#diff-f4937dc9ebc1a859941da00df7f1fd1cR23 it's 200 by default. The same for line 74

Reply:
> But we should use the settings https://github.com/neo-project/neo-modules/pull/336/files#diff-f4937dc9ebc1a859941da00df7f1fd1cR23 it's 200 by default. The same for line 74

We should make this hard code before the fork height and use the settings after that or it'll make policy confusion since we have to upgrade CN before the fork height but we don't want it go into effect until block `6216000`.